src/Scouting.elm

```diff
@@ -0,0 +1,141 @@
+module Scouting exposing (main)
+
+import Browser
+import Html
+import Html.Attributes exposing (placeholder, style, value)
+import Html.Events exposing (onClick, onInput)
+import Maybe
+import String
+
+
+main : Program () Model Msg
+main =
+    Browser.sandbox
+        { init = init
+        , view = view
+        , update = update
+        }
+
+
+type Msg
+    = NameInput String
+    | TeamInput String
+    | MatchInput String
+    | DriverStationInput String
+    | Start
+
+
+type alias Model =
+    { scouterName : String
+    , teamNum : Int
+    , matchNum : Int
+    , driverStation : String
+    , isStarted : Bool
+    }
+
+
+init : Model
+init =
+    { scouterName = ""
+    , teamNum = 0
+    , matchNum = 0
+    , driverStation = ""
+    , isStarted = False
+    }
+
+
+dropDown_driverStation : String
+dropDown_driverStation =
+    "visible"
+
+
+update : Msg -> Model -> Model
+update msg model =
+    case msg of
+        NameInput input ->
+            { model
+                | scouterName = input
+            }
+
+        TeamInput input ->
+            { model
+                | teamNum = Maybe.withDefault 0 <| String.toInt input
+            }
+
+        MatchInput input ->
+            { model
+                | matchNum = Maybe.withDefault 0 <| String.toInt input
+            }
+
+        DriverStationInput input ->
+            { model
+                | driverStation = input
+            }
+
+        Start ->
+            { model
+                | isStarted = not model.isStarted
```

Comment:
Is a toggle really what we want?

Reply:
I think I may do a 2 option view for this (not the entire view tho), where one option is 6 buttons and the other is to show the selected option/select other

Reply:
Referring to the current situation -
Is it fine that the button is a Bool?
Or did you mean that it would be better as a type alias or something?

Reply:
> Is it fine that the button is a Bool?

Yes, a button usually indicates a binary value, thus a `Bool` suits perfectly.

> Or did you mean that it would be better as a type alias or something?

`type alias`es are named this way for a reason - they are just aliases to existing types.
You don't ever really need to create a type alias, because the type already exists (`{ a : Bool }` is an existing type) - you just provide a name to be able to refer to it in a more convenient fasion.
You can certainly create a
```elm
type alias ButtonPressedState
    = Bool
```
but that's not really making anything more convenient, does it?